services/121-service/src/cronjob/cronjob.service.ts

```diff
@@ -25,7 +38,16 @@ export class CronjobService {
     const accessToken = await this.axiosCallsService.getAccessToken();
     const url = `${this.axiosCallsService.getBaseUrl()}/financial-service-providers/intersolve-voucher/cancel`;
     const headers = this.axiosCallsService.accesTokenToHeaders(accessToken);
-    await this.httpService.post(url, {}, headers);
+    const response: AxiosResponse = await this.httpService.post(
+      url,
+      {},
+      headers,
+    );
+    if (response.status < 200 || response.status >= 300) {
+      throw new Error(
+        `Failed to run Cron Job cronCancelByRefposIntersolve. Status code: ${response.status}`,
+      );
+    }
```

Comment:
Some of these cronjobs take a long time to process (expecially the visa one). So in that case a 'timout' could occur I think which has response code '0' 

If this happens this error would throw in production, while the cronjob is actually running succesfull. 

Maybe instead of this throw error response could you have the cronjob controller return an object like this in which CronjobName is a new Enum:

```
[
 {
  "cronjobName": CronjobName.validateCommercialBankEthiopiaAccountEnquiries
statusCode: 200 
},
... other cronjobs object
]
```
The api test can than validate this object. The enum cronjob names can also be used in console.log etc

Other solutions are probably also possible.